DigitalLearningSolutions.Web/ViewModels/FindYourCentre/FindYourCentreViewModel.cs

```diff
@@ -0,0 +1,19 @@
+namespace DigitalLearningSolutions.Web.ViewModels.FindYourCentre
+{
+    using DigitalLearningSolutions.Web.Helpers;
+
+    public class FindYourCentreViewModel
+    {
+        public string Url { get; set; }
+
+        public FindYourCentreViewModel()
+        {
+            Url = ConfigHelper.GetAppConfig()["CurrentSystemBaseUrl"] + "/findyourcentre?nonav=true";
+        }
+
+        public FindYourCentreViewModel(string centreId)

```

Comment:
I would have probably added a string? centreId constructor to avoid the null check in the controller, but I don't think it matters or makes anything less difficult to understand :)